### ISSUE TITLE
Menu doesn’t work with keyboard navigation

### ISSUE THREAD

**Issue by jouni** [web-padawan/polymer3-webpack-starter#39]
You can focus and toggle the menu items using the keyboard but the view doesn’t change.

**Comment by web-padawan** [comment]
Are you sure you are using latest version?
Both Lumo and Material live demos work for me:

- https://polymer3-webpack-starter.herokuapp.com

- http://polymer3-material-starter.herokuapp.com

**Comment by jouni** [comment]
Hmm, strange. Working fine now. Maybe I got some cached version yesterday 🤷‍♂️
